images_config/ks-robot-master.cfg

```diff
@@ -106,71 +106,72 @@ update-grub
 fi
 %end
 
-##### UPGRADE KERNEL #################################################
+################ UPGRADE KERNEL ################ 
 %post --interpreter=/bin/bash
 apt-get install --install-recommends linux-generic-lts-xenial -y --force-yes
 %end
 
-################## INSTALL GRAPHICAL INTERFACE ############
+################ INSTALL GRAPHICAL INTERFACE ################ 
 %post --interpreter=/bin/bash
 apt-get install lightdm -y -f
 dpkg-reconfigure lightdm
 %end
 
-####################INSTALLING ROS#######################################     
-%post --interpreter=/bin/bash
-!/bin/bash
-sh -c 'echo "deb http://packages.ros.org/ros/ubuntu trusty main" >> /etc/apt/sources.list.d/ros-latest.list'
-wget http://packages.ros.org/ros.key -O - | apt-key add - 
-apt-get update
-apt-get install ros-indigo-roscpp -y
-apt-get install ros-indigo-rosbash -y
-apt-get install ros-indigo-ros -y
-apt-get install ros-indigo-care-o-bot-robot -y
+################ ADD USERS ################ 
+%post --interpreter /bin/bash
+echo "::Adding users and groups ::"
+echo "session	required	pam_mkhomedir.so " >> /etc/pam.d/common-session
```

Comment:
probably you have the problem because the /u partition is still not created. You should configure the nfs before create the robot user https://github.com/ipa-nhg/setup_cob4/blob/ReviewInstall/images_config/ks-robot-master.cfg#L159-L166 and after that you have to mount the partition using:

`mount -a`